src/Geosuggest.jsx

```diff
@@ -319,22 +331,21 @@ class Geosuggest extends React.Component {
       input = <Input className={this.props.inputClassName}
         ref='input'
         value={this.state.userInput}
-        onChange={this.onInputChange.bind(this)}
-        onFocus={this.onInputFocus.bind(this)}
-        onBlur={this.onInputBlur.bind(this)}
         style={this.props.style.input}
-        onNext={() => this.activateSuggest('next')}
-        onPrev={() => this.activateSuggest('prev')}
-        onSelect={() => this.selectSuggest(this.state.activeSuggest)}
-        onEscape={this.hideSuggests.bind(this)} {...attributes} />,
+        onChange={this.onInputChange}
+        onFocus={this.onInputFocus}
+        onBlur={this.onInputBlur}
+        onNext={this.onNext}
+        onPrev={this.onPrev}
+        onSelect={this.onSelect}
+        onEscape={this.hideSuggests} {...attributes} />,
       suggestionsList = <SuggestList isHidden={this.state.isSuggestsHidden}
         style={this.props.style.suggests}
-        suggestItemStyle={this.props.style.suggestItem}
```

Comment:
I think this line was mistakenly deleted during the rebase.
Removing this line breaks the `style.suggestItem` prop.